script/Arcueid/Day 6/06_00_ARC06_4.txt

```diff
@@ -131,32 +158,43 @@ I nod my head in understanding and resume eating.
 [sha:8743b20797717f6430f00a7f6cd0d32df4f91a65]{
 -- Page 335, Offset 14743.
 -- 朝食と昼食を少なめにしていたおかげか、夜になって食欲が戻ってきた。
-Perhaps it was because I had eaten less breakfast and lunch, but it seems like my appetite had come back in the evening.
+Perhaps as a result of having a light breakfast followed by a light lunch,
+ by evening I had developed quite the appetite.
 }
 [sha:d47919b516ce7b6992ea0233941e363a5e1fdaf2]{
 -- Page 335, Offset 14744.
 -- ……それでも、今夜の<献立|メニュー>はやや目に痛い。
-...But with that being said, tonight's menu is rather sore to my eyes.
+// Replaced 目に痛い with EN set phrase
+...though despite that, I'm finding tonight's menu a little difficult to stomach.
 }
 [sha:dbf45b03b93a75bdc5ccb7f057db36dfb3042630]{
 -- Page 335, Offset 14745.
 -- 牛のヒレ肉やフォアグラだけでなく、真っ赤なソースが血液を連想させて、否応にも昨日までの出来事を思い返してしまう。
-It wasn't only the beef fillet and foie gras, but also the bright red sauce that reminded me of blood, which inevitably made me think back to the events of yesterday.
+Not just because of the beef tenderloin and foie gras,
+ but also because the deep red sauce forcefully reminds me of blood,
+ and by extension all of the events that led up to yesterday.
 }
 [sha:fb7862f81594d6d80ddede52bbe998bcbf21f6eb]{
 -- Page 336, Offset 14746.
 -- 突然の殺人。ありえない再会。ありえない怪物。
-The sudden murders. The unlikely reunion. The unbelievable monster.
+A freak murder. An impossible reunion. An unbelievable monster.
```

Comment:
Is "freak murder" right? I think of "freak" as being uncharacteristic or anomalous, not "abrupt". Maybe just "An unexpected murder"